skopt/optimizer/optimizer.py

```diff
@@ -195,6 +196,7 @@ def tell(self, x, y, fit=True):
             only be fitted after `n_random_starts` points have been queried
             irrespective of the value of `fit`.
         """
+        is_categorical = all([isinstance(dim, Categorical) for dim in self.space.dimensions])
```

Comment:
I did not realize you were commenting on the `optimizer` diff. I've addressed this.